sdk/azcore/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/Azure/azure-sdk-for-go/sdk/azcore
 
 require (
-	github.com/Azure/azure-sdk-for-go/sdk/internal v0.3.0
-	golang.org/x/net v0.0.0-20200904194848-62affa334b73
+	github.com/Azure/azure-sdk-for-go/sdk/internal v0.5.0
+	golang.org/x/net v0.0.0-20201010224723-4f7140c49acb
```

Comment:
I'm surprised this showed up.  Is it new in `internal`?

Reply:
Yes it's included because of the http2 test, it can be removed in a future change now that I added WithHTTP2Enabled in the package